src/config/development.ts

```diff
@@ -9,6 +9,8 @@ import { gwei2wei } from '@/helpers/blockchain';
 
 const INFURA_API_KEY = process.env.NEXT_PUBLIC_INFURA_API_KEY;
 const BASE_ROUTE = 'https://serve.giveth.io';
+// IMPORTANT: Using dev to make it work until staging is merged
+// const BASE_ROUTE = 'https://dev.serve.giveth.io';
```

Comment:
Can I test it on staging or it's still on the dev branch?

Reply:
still on develop, gotta pay attention and coordinate with this [pr](https://github.com/Giveth/impact-graph/pull/531)